tools/generators/workspace-generator/index.ts

```diff
@@ -41,10 +41,9 @@ function addFiles(host: Tree, options: NormalizedSchema) {
     tmpl: '',
   };
 
-  generateFiles(
-    host,
-    path.join(__dirname, 'files'),
-    path.join(options.paths.generators, options.name),
-    templateOptions,
-  );
+  const projectRoot = path.join(options.paths.generators, options.name);
+
+  generateFiles(host, path.join(__dirname, 'files'), projectRoot, templateOptions);
```

Comment:
nit: For consistency, the instances of `host` in this file should maybe be changed to `tree` as well.

Reply:
good catch! I'll update that